hnsecure/pom.xml

```diff
@@ -217,6 +220,11 @@
               <transformers>
                 <transformer implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
                   <mainClass>ca.bc.gov.hlth.hnsecure.HnsEsbMainMethod</mainClass>
+                  <manifestEntries>
+					<Name>ca/bc/gov/hlth/hnsecure/</Name>
+					<Implementation-Version>${project.version}-${BRANCH_NAME}-${BUILD_NUMBER}</Implementation-Version>	
+                  	<Git-Commit>${git-commit}</Git-Commit>
```

Comment:
Isn't this just pulling the hardcoded value SNAPSHOT from the property git-commit?
Edit: I see how it's populated in the build. I'd be inclined to just default it to blank. SNAPSHOT doesn't mean much on it's own.